compiler/src/iree/compiler/API/Internal/IREEReduceToolEntryPoint.cpp

```diff
@@ -47,19 +48,28 @@ static LogicalResult ireeReduceMainFromCL(int argc, char **argv,
 
   llvm::cl::OptionCategory ireeReduceCategory("iree-reduce options");
 
-  static llvm::cl::opt<std::string> testScript(cl::Positional, cl::Required,
-                                               cl::desc("<test script>"),
-                                               cl::cat(ireeReduceCategory));
+  llvm::cl::opt<std::string> testScript(cl::Positional, cl::Required,
+                                        cl::desc("<test script>"),
+                                        cl::cat(ireeReduceCategory));
 
-  static cl::opt<std::string> inputFilename(
-      cl::Positional, cl::desc("<input file>"), cl::init("-"),
-      llvm::cl::cat(ireeReduceCategory));
+  cl::opt<std::string> inputFilename(cl::Positional, cl::desc("<input file>"),
+                                     cl::init("-"),
+                                     llvm::cl::cat(ireeReduceCategory));
 
-  static cl::opt<std::string> outputFilename(
+  cl::opt<std::string> outputFilename(
       "o", cl::desc("Output filename for the reduced test case."),
       cl::value_desc("filename"), cl::init("-"),
       llvm::cl::cat(ireeReduceCategory));
 
+  cl::opt<bool> useByteCodeForTesting(
```

Comment:
capitalization nit: `Bytecode`, without capitalizing the `C`
https://mlir.llvm.org/docs/BytecodeFormat/
https://en.wikipedia.org/wiki/Bytecode

(throughout all files)